openlibrary/core/models.py

```diff
@@ -7,13 +7,11 @@
 import iptools
 from infogami.infobase import client
 
-import helpers as h
-
 #TODO: fix this. openlibrary.core should not import plugins.
 from openlibrary import accounts
 from openlibrary.utils import extract_numeric_id_from_olid
 from openlibrary.plugins.upstream.utils import get_history
-from openlibrary.core.helpers import private_collection_in
+from openlibrary.core import helpers as h
```

Comment:
Why not fix up all the relative imports at the same time? There are some more relative ones a few lines below which would make this

```
from openlibrary.core import cache, helpers as h, iprange, inlibrary, loanstats, waitinglist
from openlibrary.core.lists.model import ListMixin
```
`Seed` and `lending` are unused.

Also sorting the inputs would make them a lot easier to read.

Reply:
As we discussed on last week’s call, when I make complicated changes then reviews do not happen.  PRs like #2887, #2888, and #2897 make significant Python 3 progress but are super difficult to review.  We agreed in that discussion that my PRs should contain just one or two files and should make the minimal changes required to get the Python 3 pytests to pass.

On unused imports, `flake8 . --count --select=F401 --show-source --statistics` currently returns 420 of them in this repo.  It would be much easier to review and safer to have a series of PRs that only attack this problem because some imports have side effects.

On sorting imports, `isort` will automate this so that can be deferred to later PRs because several kinds of changes makes the PR difficult to review.

Perhaps we need pre-commits like https://github.com/cclauss/Ten-lines-or-less/blob/master/.pre-commit-config.yaml so that reviewers are used to seeing these kind of changes.